### ISSUE TITLE
Golgi cell synapse confusion

### ISSUE THREAD

**Issue by Helveg** [dbbs-lab/models#2]
After dbbs-lab/dbbs-models#29 there are now 2 AMPA synapse types: `AMPA_PF` and `AMPA_MF`, could you @SteMasoli make sure that they are placed on the right section types? I see that the model still refers to the general 'AMPA' synapse in all cases.

Could you also tell me in the following connection models which synapse they should form? Is this correct?

```
        "glomerulus_to_golgi": {
          "synapse": "AMPA_MF"
        },
        "ascending_axon_to_golgi": {
          "synapse": "AMPA_MF"
        },
        "parallel_fiber_to_golgi": {
          "synapse": "AMPA_PF"
        },
```

**Comment by SteMasoli** [comment]
Golgi cell 1 sections for Parallel fiber synpases.


if sec_index >= 4 and sec_index <= 15 or sec_index >= 18 and sec_index <= 32 or sec_index >= 42 and sec_index <= 83 or sec_index >= 85 and sec_index <= 104:

mod file PF, stellata and Purkinje.

Golgi cell 1 section for Mossy fiber and ascending axon.

if sec_index >= 108 and sec_index <= 112 or sec_index >= 114 and sec_index <= 121 or sec_index >= 128 and sec_index <= 129 or sec_index >= 131 and sec_index <= 132 or sec_index >= 135 and sec_index <= 140 or sec_index >= 144 and sec_index <= 145 or sec_index >= 147 and sec_index <= 150:

(MOD files, ampa PF Golgi, Purkinje, stellata for the asce ngin axon. MOD file used in the granule cells for the mossy fibers)

To summarize.

Golgi AMPA_PF/AA - PC, SC, BC, Golgi are the same mod with different "external" values".
Golgi AMPA_MF - The MOD is the one for the granule cell and it is valid only for MF-GrC and MF-GoC

Golgi NMDA_AA and NMDA_MF - Stellate NMDA

Granule cell NMDA - Only used for granule cells

**Comment by Helveg** [comment]
Closed by #5
